### PR TITLE
Implement shop 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, there is no shop implemented. #31 
- Currently, the footer does not fit the theme. #27 
- Currently, the website has countless little fields of interest which require improvement.

**What is the new behavior (if this is a feature change)?**
- The shop has been implemented. #31 
- The footer has been improved. #27 
- The Sub and Blackwolf components have been updated.
- The FAQ has been updated.
- The navigation icon is now the button to launch the shop cart.
- Types in the dummy data have been fixed.
- The code quality has been improved by enforcing our guidelines.

**Other information**:
- The version has been updated to v0.3.0
[Learn more about our versioning](https://github.com/aichner/React-MDB-Template#versioning)

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks good to me.

File: src/components/molecules/Cart/index.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  MDBBtn,
+  MDBIcon,
+} from 'mdbreact';
+
+//> CSS
+import './cart.scss';
+
+class Cart extends React.Component {
+  constructor(props) {
+  super(props);
```
Comment (schettn) [inline src/components/molecules/Cart/index.jsx]: The indents do not look right. Please correct them.

**Review by schettn — CHANGES_REQUESTED** [review]
Should be ok to merge after one change.

File: src/components/molecules/Cart/index.jsx (hunk 1 above)
Comment (kleberbaum) [inline src/components/molecules/Cart/index.jsx]: There should be some kind of indentation.

**Review by schettn — APPROVED** [review]
Should be ok to merge.

File: src/components/organisms/Sections/Shop/index.jsx [hunk 2]
```diff
@@ -6,107 +6,116 @@ import React from 'react';
...
+      return true;
+    });
+
+    this.setState({
+      products: res
+    });
+  }
+
+  render(){
+
```
Comment (kleberbaum) [inline src/components/organisms/Sections/Shop/index.jsx]: Please strip all unnecessary newline.

File: src/components/organisms/Sections/BlackWolf/index.jsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  }
+
+  onClick = (nr, value) => {
+    this.setState({
+      ["step"+nr]: value
+    });
+  };
+
+  render(){
+
```
Comment (kleberbaum) [inline src/components/organisms/Sections/BlackWolf/index.jsx]: Please strip all unnecessary newline.

File: src/utilities/checkout.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+      checkout: res.data.checkoutCustomerAssociate.checkout,
+      isCustomerAuthOpen: false
+    });
+  });
+}
+
+/** 
+ * SPDX-License-Identifier: (EUPL-1.2)
+ * Copyright © 2019 Werbeagentur Christian Aichner
+ */
```
Comment (kleberbaum) [inline src/utilities/checkout.js]: Please end every source file with a newline.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**